package.json

```diff
@@ -1,9 +1,15 @@
 {
   "private": true,
-  "workspaces": [
-    "packages/*",
-    "examples/*"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/*",
+      "examples/*"
+    ],
+    "nohoist": [
+      "**/html-webpack-plugin",
```

Comment:
hmm, i guess there's two versions of html-webpack-plugin (one from storybook and one from webpack-config-anansi) but kinda weird this change doesn't cause the yarn.lock to change.

Reply:
yarn.lock specifies resolutions, which already has a distinct piece. this specifies where the files go in the directory path.